.github/workflows/base-release.yaml

```diff
@@ -133,8 +135,8 @@ jobs:
         with:
           distribution: goreleaser-pro
           version: latest
-          args: continue --merge --timeout 2h
+          args: continue --merge --timeout 2h --id ${{ inputs.distribution }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          COSIGN_YES: true
```

Comment:
Can you double-check this? I think my version is older than what we have in this repo here when it comes to cosign.